docs/programming-oak.md

```diff
@@ -2,18 +2,19 @@
 
 This document walks through the basics of programming in Oak.
 
-- [Writing an Oak Node](#writing-an-oak-node)
-  - [Per-Node Boilerplate](#per-node-boilerplate)
-  - [Generated gRPC service code](#generated-grpc-service-code)
-- [Using an Oak Application from a client](#using-an-oak-application-from-a-client)
-  - [Connecting to the Oak Manager](#connecting-to-the-oak-manager)
-  - [Starting the Oak Application](#starting-the-oak-application)
-- [gRPC Request Processing Path](#grpc-request-processing-path)
-- [Node Termination](#node-termination)
-- [Nodes, Channels and Handles](#nodes-channels-and-handles)
-- [Persistent Storage](#persistent-storage)
-- [Testing](#testing)
-  - [Testing Multi-Node Applications](#testing-multi-node-applications)
+- [Programming Oak](#programming-oak)
```

Comment:
note it is not common to include the title in the TOC, or was this generated by some automated tool?

Reply:
I didn't touch this file, it was changed by the `format`.

Reply:
From within Docker? But why did it change it now?

Reply:
Fwiw `[Programming Oak](#programming-oak)` is an addition, perhaps from a merge or by accident?

Reply:
I'll revert this change then

Reply:
Oh, I get it
This change is automatically applied by VSCode, when I save the file.